cantabular/static_dataset.go

```diff
@@ -153,3 +153,91 @@ func (c *Client) StaticDatasetQueryStreamCSV(ctx context.Context, req StaticData
 	// Stream is responsible for closing the response body
 	return rowCount, stream.Stream(ctx, res.Body, transform, consume)
 }
+
+// Checks the number of observations returned from a cantabular query
+func (c *Client) CheckQueryCount(ctx context.Context, req StaticDatasetQueryRequest) (int, error) {
```

Comment:
Is there a unit test for this?

Reply:
This is fully tested as part of component tests in dp-population-types-api that consumes it.